akvo/rsr/spa/app/modules/project-view/project-view.jsx

```diff
@@ -26,19 +28,25 @@ const _Header = ({ title, projectId, publishingStatus, hasHierarchy, userRdr, jw
   const { t } = useTranslation()
   const showEnumerators = role !== 'enumerator' && (isRSRTeamMember(userRdr) || shouldShowFlag(userRdr.organisations, flagOrgs.ENUMERATORS))
   const disableResults = publishingStatus !== 'published'
-  const showResultAdmin = isRSRTeamMember(userRdr)
+  const showResultAdmin = shouldShowFlag(userRdr.organisations, flagOrgs.AKVO_USERS) || (getSubdomainName() === 'rsr4')
+  const labelResultView = showResultAdmin ? 'Results Overview' : 'Results'
   return [
     <header className="main-header">
       {(!jwtView && prevPathName != null) && <Link to={prevPathName}><Icon type="left" /></Link>}
       <h1>{title ? title : t('Untitled project')}</h1>
     </header>,
     !jwtView &&
-    <Route path="/projects/:id/:view?" render={() => {
+    <Route path="/projects/:id/:view?" render={({ match: { params: { view } } }) => {
+      const _view = sections.findIndex(it => it.key === view) !== -1 ? 'editor' : view
+      const tabsProps = showResultAdmin ? { defaultActiveKey: 'results' } : { defaultActiveKey: 'results', activeKey: _view }
+      if(_view === 'results' && shouldShowFlag(userRdr.organisations, flagOrgs.NUFFIC)){
```

Comment:
This exception would only need to be on `production`, but we can make the change in a later deploy if we want to.